Documentation/Signoffs/OEM Component- Auto_Light-signoff.md

```diff
@@ -0,0 +1,99 @@
+
+However, the design with only a voltage divider has a potential risk due to the chance of a voltage spike reaching the microcontroller and interference from other signals. This design shows the vulnerability to signal interferece through the shared ground terminal and the ability for the current to backfeed onto another divider. The voltage divider also allows the chance of the signal voltage to exceed the limit of 5 V due to a spike in the input voltage on the lamp node. The design was modified to include optoisolators to remidy the hazards of both constraints as shown in Figure 2. The optoisolator will take reduced voltage from the Nissan Vehicle and convert it into a light signal. Then a photosensor will detect the light signal and allow the 5 V to pass through the pin of the microcontroller. The LED that is used to trigger the photosensor also prevents any backfeed current from false triggering a signal. This isolates both systems so that the voltage to the lights does not have a direct path to the signal input. With the microcontroller being used for the 5 V supply in the signal, there should not be a voltage spike to the controller since it would have to be generated by the controller. 
+
+## Analysis
+-----
+From the Datasheet
+
+![LVT 817](https://user-images.githubusercontent.com/101354063/206586290-37ae6b0b-b3ca-41a7-96c7-75b4382fc6b4.PNG)
+
+Forward Current = 40 mA  for opto-isolation [3]
+
+The Voltage divider will need to produce a maximum output 5 V and 40 mA.
+
+### Voltage divider
+
+Vout = Vin ${ R2 \over R1+R2}$
+
+5V = 12V ${ 125 \ ohms \over 250 \ ohms  + 125 \ ohms}$
+
+$Total \ Resistance: 375 \ ohms = {250 \ ohms +125 \ ohms}$ 
+
+$Max \ current: 40\ mA = {10V \over 375 \ ohms }$
+
+$Min \ current: 26.67\ mA = {10V \over 375 \ ohms }$
+
+Test different voltages going through Voltage Divider(0 to 15)
+
+![image](https://user-images.githubusercontent.com/101354063/206635438-b9d504b8-7e54-471c-8ddc-083a5be3e416.png)
+
+Test different voltages going through Voltage Divider(10 to 15)
+
+![image](https://user-images.githubusercontent.com/101354063/206637463-7d53790e-b779-426a-9310-c4f548f22864.png)
+ 
+## Master microcontroller communication
```

Comment:
If this is the case, then this schematic needs to be combined with the other schematics which depict the master MCU